_posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md

```diff
@@ -0,0 +1,97 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+From the beginning, everyone was very open-minded, friendly, helpful, and patient.
+They never made me feel uncomfortable, even though I may have asked some annoying questions (especially for the developers).
+This way the working atmosphere has been pleasant from the first day on.
+
+The team is quite young and very international.
+It is fun to work with so many different nationalities and communicate in different languages.
+At lunch time we eat together, enjoy the sun in the park (yes, there are sunny days in Hamburg!{% emoji wink %}) or grab a coffee.
+This way I've been already shown many nice and delicious restaurants (my favorite one is a Chinese one) and got some insider tips for my first days in Hamburg.
+
+## The company events
+
+Communication and transparency is really important here at ePages.
+To be always up-to-date, we have regular team meetings which gave me the possibility to get an overview of what is going on and who's working at which topic.
+
+Besides these weekly meetings we have some "special events" such as the [ePages Hackathon](https://developer.epages.com/blog/2017/03/29/epages-hackathon-2017.html) or the ePages YOU.
+On our annual Hackathon event in March I got to know many other ePagees from Barcelona, New York, and Jena.
+As it was my first Hackathon, it was a really great experience for me and it was nice to meet a lot of new people.
+
+##  The tasks
+
+Making coffee and copies and doing all the annoying stuff nobody else wants to do?
+That's how an internship could be... but not at ePages.
+Taking into account my wishes and interests, I got my own varied and interesting tasks.
+For example, I had the chance to research on some software localization tools such as [PhraseApp](https://phraseapp.com/) or I was even engaged in making a [video of our Hackathon](https://www.youtube.com/watch?v=j-qMUo8_ChQ).
+I am really happy, that I got my own responsible tasks and felt like a "real" colleague and not only like an intern.
+
+## The secret recipe
+
+It is difficult to determine THE one reason why ePages made me feel welcome so fast.
+Let me present my top three:
+
+1. **Great colleagues** lead to a great working atmosphere.
+2. A structured and detailed [**onboarding**](https://developer.epages.com/blog/2016/08/04/5-steps-to-make-you-feel-comfortable-in-your-new-job.html) simplifies the start tremendously.
+3. Undertaking **responsible tasks** makes you feel like part of the team.
+
+After so many positive aspects, it won't surprise you that I decided to write my master thesis here at ePages after I finished my internship.
```

Comment:
...after I will finish my internship. 

The internship has not ended yet, has it?